index.html

```diff
@@ -814,150 +814,156 @@ <h2>Privacy Considerations</h2>
 
     <section class="appendix">
       <h2>Test Vectors</h2>
-      <p>
-The following test vectors are provided to assist with implementers.
-      </p>
-
-      <pre class="example">
+      <section>
+        <h3>Representation: Ed25519Signature2020</h3>
+        <p>
+The signer needs to generate a private/public key pair with the private key used 
+for signing and the public key made available for verification. Below the 
+[[MULTIBASE]]/[[MULTICODEC]] representation for the public key, <code>ed25519-pub</code>,
+and the representation for the private key, <code>ed25519-priv</code>, are shown.
```

Comment:
Shifted to (American) English word order.
```suggestion
for signing and the public key made available for verification. The 
[[MULTIBASE]]/[[MULTICODEC]] representation for the public key, <code>ed25519-pub</code>,
and the representation for the private key, <code>ed25519-priv</code>, are shown below.
```